docs/nextflow/reduce-api-calls.md

```diff
@@ -0,0 +1,110 @@
+---
+title: Reducing Wave API calls
+description: Learn how to configure Nextflow to freeze containers and reduce API calls
+date created: 2025-09-30
+date edited: 2025-10-31
+tags: [nextflow, wave, rate limits, guides]
+---
+
+Large-scale pipelines that pull container images across thousands of concurrent tasks can encounter Wave rate limits. This guide describes how to configure your Nextflow pipeline to use Wave freeze and reduce API calls to avoid rate limits.
+
+:::note
+Wave applies rate limits to container builds and pulls (manifest requests). Authenticated users have higher rate limits than anonymous users. See [API limits](../api.md#api-limits) for more information.
+:::
+
+## How Wave freeze avoids rate limits
+
+Wave freeze builds your container image once and stores it in your registry. After the initial build, the source of the container image manifest and layers are redirected to your private registry by Wave.
```

Comment:
I Quite like the freeze description we have in the wave Q&A Draft google doc as it covers some additional value adds of using wave freeze which may remove the need to draw comparison between wave request flows (Build , Augment , Freeze & Mirror)